NSS-Wacky-Eels/WorkforceManagement
# Af create employee

## Link to Ticket
Closes # [Link](https://github.com/NSS-Wacky-Eels/WorkforceManagement/issues/2)

## Description of Proposed Changes
- Fixed EmployeeAddEditView model so dropdown box values will correctly correspond with departments when creating an employee in the EmployeeView. 
- Added connection string to Config.cs file in the Tests Repository. This enables me to run the test that properly creates the employee. 

## Steps to Test

- Try to add an employee by clicking "Create New" on the index page of Employee. Fill out the form with data, and make sure the newly created employee has been added to the index page. 
- Run "Post_CreateAddsEmployee" test in the test explorer. 

Outline the steps to test

```
git fetch --all
git checkout <af-createEmp>
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* EmployeeController
* EmployeeTests
* Config.cs

## Mentions @username

@KaylaReid
@alexfont321
@thatmikeparrish
@taylorlgulley


## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
